src/utils/diff/diff-colors.js

```diff
@@ -0,0 +1,22 @@
+const colors = {
```

Comment:
Rewrite the new files with TypeScript.
Remove prefix `diff` for all nested files in the `src/utils/diff/` folder.